test/evaluation.jest.js

```diff
@@ -357,7 +356,7 @@ describe('Page.evaluate', function() {
     });
     expect(result).toEqual([42]);
   });
-  it.fail(WEBKIT)('should not throw an error when evaluation does a synchronous navigation and returns an object', async({page, server}) => {
+  (WEBKIT ? it.skip : it)('should not throw an error when evaluation does a synchronous navigation and returns an object', async({page, server}) => {
```

Comment:
I think this is ok for experimenting, but it would be nice to figure out the ergonomics.